CHANGES_NEXT_RELEASE

```diff
@@ -5,5 +5,6 @@
        the new CLI parameter -mutexTimeStat is set (Issue #911)
 - Fix: Removing trailing slashes for path of URI before treating the request (Issue #828)
 - Fix: Fixed a bug regarding subscriptions with empty entity-type (Issue #691)
-- Fix: updateContext propagated to the entire Service Path subtree, instead of just scoping to the particular Service Path in the Fiware-Service header (Issue # 885)
-- Mongo driver migrated to legacy-1.0.2
+- Fix: updateContext propagated to the entire Service Path subtree, instead of just scoping to the particular Service Path in the Fiware-Service header (Issue #885)
+- Fix: Mongo driver migrated to legacy-1.0.2
```

Comment:
Not sure if this is actually a fix of a "evolution" ;) Not very important anyway, let's keep as it is.

NTC